feature/gribi/ate_tests/base_hierarchical_route_installation_test/README.md

```diff
@@ -0,0 +1,89 @@
+# TE-3.1: Base Hierarchical Route Installation
```

Comment:
This file should probably be named base_hierarchical_route_installation_test.md.  The [contribution guide](https://github.com/openconfig/featureprofiles/blob/main/CONTRIBUTING.md#test-suite-organization) mentions that test documentation should be named the test name, not README.

Reply:
Aah. This is intentional. I changed the format to README.md because it shows up in the folder without the need to open the file. I have discussed this with Likai and will be making changes to the existing test readmes as well as the contributing guide.